src/components/session/SessionsStateFilter.vue

```diff
@@ -0,0 +1,62 @@
+<script setup>
+import { defineProps } from "vue";
+
+const props = defineProps({
+  sessions: {
+    type: Array,
+    default: null,
+  },
+  numberOfOpenClosedSessions: {
+    type: Object,
+    default: null,
+  },
+  sessionSelectedState: {
+    type: String,
+    default: "open",
+  },
+  changeSessionState: Function,
+});
+</script>
+<template>
+  <v-btn
+    prepend-icon="mdi-lock-open-variant-outline"
```

Comment:
Please change the icons to the ones GitHub uses:
![image](https://github.com/user-attachments/assets/f6f81398-7a22-4444-ad25-555aa9203ae9)

Especially the "closed lock" icon was giving the impression that this might be "closed source" or "secret", and the checkmark as used by GitHub gives more of a "done" feeling to the closed PRs.

Reply:
Added it like this - 
![CleanShot 2025-04-24 at 15 30 24@2x](https://github.com/user-attachments/assets/975d5b38-2f09-43d5-98c0-5d43a03666d5)